test/upgrade/continual.go

```diff
@@ -38,6 +40,9 @@ func ContinualTest() pkgupgrade.BackgroundOperation {
 			// overwrite configuration
 			config.FailOnErrors = true
 			config.Interval = 10 * time.Millisecond
+			config.BrokerOpts = append(config.BrokerOpts, resources.WithBrokerClassForBrokerV1Beta1(v1beta1.BrokerClass))
+			config.FinishedSleep = 40 * time.Second
+			config.ConfigTemplate = "../../../../../../test/upgrade/config.toml"
```

Comment:
maybe we can just use `../config.toml`

Reply:
No, it is relative to the `eventing/test` directory in vendor directory. This is in `knaitve/test`.